vyxal/main.py

```diff
@@ -9,6 +9,7 @@
 import types
 
 import vyxal.encoding
+import itertools
```

Comment:
What's this for?

Reply:
Used it in a previous version of code, but forgot to remove it
```suggestion
```